src/core/public/http/types.ts

```diff
@@ -262,7 +262,7 @@ export interface HttpFetchOptions extends HttpRequestInit {
    * When `true`, if the response has a JSON mime type, the {@link HttpResponse} will use an alternate JSON parser
    * that converts long numerals to BigInts. Defaults to `false`.
    */
-  withLongNumerals?: boolean;
+  withLongNumeralsSupport?: boolean;
```

Comment:
If I'm not mistaken, this is part of core's public API. Are we concerned with this being breaking?

Reply:
Yeah; i did this when 2.11 was not released. I updated the code and marked it deprecated. I reintroduced the consumption of it and added it to the deprecation changelog.